### PR TITLE
CORE-6704 - updated KDocs after review by tech writers

### PR DESCRIPTION
**Pull request by alexey-kadyrov-r3**

### CODE REVIEW

File: cipher-suite/src/main/kotlin/net/corda/v5/cipher/suite/ConfigurationSecrets.kt [hunk 1]
```diff
@@ -2,8 +2,7 @@ package net.corda.v5.cipher.suite
 
 /**
  * Provides ways of getting the secret value for the [CryptoServiceProvider] from the service configuration.
- * The secret value have to be declared as Map<String, Any> (or Map<String, Object>) in the service
- * configuration class for that to work.
+ * The secret value must be declared as Map (or Map) in the service configuration class.
```
Comment (nadinequinr3) [inline cipher-suite/src/main/kotlin/net/corda/v5/cipher/suite/ConfigurationSecrets.kt:5]: Is this correct?

File: cipher-suite/src/main/kotlin/net/corda/v5/cipher/suite/CryptoServiceProvider.kt [hunk 2]
```diff
@@ -16,10 +16,12 @@ interface CryptoServiceProvider<T : Any> {
...
 
     /**
+     * Creates a new instance of the [CryptoService] implementation.
+     *
      * @param config crypto service configuration
      * @param secrets provides access to decrypting the configuration secrets
      *
-     * The secrets have to be declared as Map<String, Any> in the corresponding POJO, the JSON will look like
+     * The secrets have to be declared as Map in the corresponding POJO, the JSON will look like
      * in the example bellow for the property called 'passphrase'
```
Comment (nadinequinr3) [inline cipher-suite/src/main/kotlin/net/corda/v5/cipher/suite/CryptoServiceProvider.kt]:
```suggestion
     * the example below for the property called 'passphrase'
```

File: cipher-suite/src/main/kotlin/net/corda/v5/cipher/suite/CryptoServiceUtils.kt [hunk 3]
```diff
@@ -7,18 +7,18 @@ import net.corda.v5.crypto.HMAC_SHA256_ALGORITHM
...
+ * transforms it to HEX, and takes the first 12 characters of that result.
  *
  * @param tenantId The tenant's id which the [alias] belongs to
  * @param alias Alias as supplied by the [tenantId].
  * @param secret Secret.
  *
- * @return computed alias which must be unique and must be deterministic, e.g. for the same
- * inputs ([tenantId] and [alias]) always produce the same output.
+ * @return A computed alias that must be unique and deterministic. The same inputs ([tenantId] and [alias])
+ * always produce the same output
```
Comment (nadinequinr3) [inline cipher-suite/src/main/kotlin/net/corda/v5/cipher/suite/CryptoServiceUtils.kt]:
```suggestion
 * always produce the same output.
```

File: cipher-suite/src/main/kotlin/net/corda/v5/cipher/suite/CryptoServiceUtils.kt [hunk 4]
```diff
@@ -29,19 +29,19 @@ fun computeHSMAlias(
...
  *
  * @param tenantId The tenant's id which the [alias] belongs to
  * @param alias Alias as supplied by the [tenantId].
  * @param secret Secret.
  * @param take How mane characters to take from the result, minimum is 12, maximum 32
  *
- * @return computed alias which must be unique and must be deterministic, e.g. for the same
- * inputs ([tenantId] and [alias]) always produce the same output.
+ * @return A computed alias that must be unique and deterministic. The same inputs ([tenantId] and [alias])
+ * always produce the same output
```
Comment (nadinequinr3) [inline cipher-suite/src/main/kotlin/net/corda/v5/cipher/suite/CryptoServiceUtils.kt]:
```suggestion
 * always produce the same output.
```

File: cipher-suite/src/main/kotlin/net/corda/v5/cipher/suite/CustomSignatureSpec.kt [hunk 5]
```diff
@@ -23,13 +23,30 @@ import java.security.spec.AlgorithmParameterSpec
...
+     * Signature parameters.
+     */
     val params: AlgorithmParameterSpec?
 ) : SignatureSpec(signatureName) {
 
+    /**
+     * Creates an instance of the [CustomSignatureSpec] with specified signature name and the digest name,
+     * [params] are set to null value.
+     *
+     * @param signatureName a signature-scheme name as required to create [java.security.Signature]
```
Comment (nadinequinr3) [inline cipher-suite/src/main/kotlin/net/corda/v5/cipher/suite/CustomSignatureSpec.kt]:
```suggestion
     * @param signatureName A signature-scheme name as required to create [java.security.Signature]
```

File: cipher-suite/src/main/kotlin/net/corda/v5/cipher/suite/schemes/KeySchemeCapability.kt [hunk 6]
```diff
@@ -1,9 +1,16 @@
 package net.corda.v5.cipher.suite.schemes
 
 /**
- * A key usages, such as for signing or deriving a shared secret or any combination.
+ * Defines which operations the key scheme cna be used for, such as for signing or/and deriving a shared secret.
```
Comment (nadinequinr3) [inline cipher-suite/src/main/kotlin/net/corda/v5/cipher/suite/schemes/KeySchemeCapability.kt]:
```suggestion
 * Defines which operations the key scheme can be used for, such as for signing or/and deriving a shared secret.
```

**Review by nadinequinr3 — CHANGES_REQUESTED** [review]

File: cipher-suite/src/main/kotlin/net/corda/v5/cipher/suite/ConfigurationSecrets.kt (hunk 1 above)
Comment (alexey-kadyrov-r3) [reply cipher-suite/src/main/kotlin/net/corda/v5/cipher/suite/ConfigurationSecrets.kt:5]: yep

**Review by nadinequinr3 — APPROVED** [review]

**Review by driessamyn — APPROVED** [review]
LGTM

**Pull request merged**